lib/studies/shrine/login.dart

```diff
@@ -59,7 +59,6 @@ class LoginPage extends StatelessWidget {
               ),
             )
           : Scaffold(
-              appBar: AppBar(backgroundColor: Colors.white),
```

Comment:
Why was this removed?

Reply:
Same as for Rally, no need for it (and there's already the back button)